app/views/support_interface/ucas_matches/index.html.erb

```diff
@@ -1,35 +1,5 @@
 <%= render 'support_interface/candidates/candidates_navigation', title: 'UCAS matches' %>
 
-<table class="govuk-table">
-  <thead class="govuk-table__head">
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">Status</th>
-      <th scope="col" class="govuk-table__header">Candidate</th>
-      <th scope="col" class="govuk-table__header">Last update</th>
-    </tr>
-  </thead>
-
-  <tbody class="govuk-table__body">
-    <% @matches.each do |match| %>
-      <tr class="govuk-table__row">
-        <td class="govuk-table__cell">
-          <%= render TagComponent.new(text: match.matching_state.humanize, type: match.processed? ? :green : :purple) %>
-
-          <% if match.invalid_matching_data? %>
-            <%= render TagComponent.new(text: "Invalid data", type: :red) %>
-          <% elsif match.action_needed? %>
-            <%= render TagComponent.new(text: "Action needed", type: :yellow) %>
-          <% end %>
-        </td>
-
-        <td class="govuk-table__cell">
-          <%= govuk_link_to match.candidate.email_address, support_interface_ucas_match_path(match) %>
-        </td>
-
-        <td class="govuk-table__cell">
-          <%= match.updated_at.to_s(:govuk_date_and_time) %>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<%= render PaginatedFilterComponent.new(filter: @filter, collection: @matches) do %>
```

Comment:
I can't believe we have a reusable component that's this big AND is still useful! 🎉